extensions/wikia/AdEngine/js/video/player/porvata/porvataPlayerFactory.js

```diff
@@ -40,6 +41,10 @@ define('ext.wikia.adEngine.video.player.porvata.porvataPlayerFactory', ['wikia.l
 					width = newWidth;
 					height = newHeight;
 				}
+				if (!width || !height) {
+					width = params.container.offsetWidth;
+					height = params.container.offsetHeight;
+				}
```

Comment:
Player size cannot be 0/0 - it will break ad load. Play method is being called without arguments: 
https://github.com/Wikia/app/blob/1afee07902b184ce5b836b2553b974729b869d61/extensions/wikia/AdEngine/js/video/player/porvata/porvata.js#L53